.gitignore

```diff
@@ -15,7 +15,6 @@ purescript/output
 
 # proxy directories
 fantasy-land
```

Comment:
u might want to remove this whole section of `.gitignore` - `cherry-pick clean` should just remove those after the `publish` so maybe not worth keeping this list here